prefab/prefab-publishing/mylibrary/build.gradle

```diff
@@ -39,14 +39,27 @@ android {
         }
     }
 
+    // Enable creation of Prefab packages and integration into AAR.
```

Comment:
Enable generation of Prefab packages and include them in the library's AAR.

Reply:
done